tests/ImageSharp.Tests/Formats/Tiff/TiffMetadataTests.cs

```diff
@@ -153,10 +156,10 @@ public void BaselineTags<TPixel>(TestImageProvider<TPixel> provider)
                 Assert.Equal("This is Название", frame.ImageDescription);
                 Assert.Equal("This is Изготовитель камеры", frame.Make);
                 Assert.Equal("This is Модель камеры", frame.Model);
-                Assert.Equal(new uint[] { 8 }, frame.StripOffsets);
+                TiffTestUtils.Compare(new Number[] { 8 }, frame.StripOffsets);
```

Comment:
Use an `IEqualityComparer` or provide a built in comparison method. Users should be able to compare instances.

Reply:
Done!